src/main/resources/react/src/components/pages/FormRegister/navbar_register/NavBarRegister.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { NavLink } from "react-router-dom";
+import cls from "./registerTabs.module.scss";
+
+const NavBarRegister = () => {
+  return (
+    <div className={cls.tabs}>
+      <NavLink to="/registration/" exact activeClassName={cls.active}>
+        Вход
+      </NavLink>
+      <NavLink to="/registration/register" activeClassName={cls.active}>
```

Comment:
It's better to create global constant and set it for links so you could change it only once instead of seeking all duplicates into the code. Your routesRegister.js uses the same paths as well.

Reply:
That is how it should be